src/main/java/org/craftercms/studio/impl/v1/service/deployment/DeploymentServiceImpl.java

```diff
@@ -381,13 +383,15 @@ private List<PublishRequest> createDeleteItems(String site, String environment,
                             item.setCommitId(commitId);
                         } else {
                             if (StringUtils.isNotEmpty(commitId)) {
-                                logger.warn("Commit ID is NULL for content " + path +
-                                        ". Was the git repo reset at some point?" );
+                                logger.warn("The item in site '{}' path '{}' has a null commit ID. Was the git " +
```

Comment:
Is this message correct? Seems like the opposite of what the "if" condition is checking

Reply:
It's correct, just confusing code. Added a clarification.